src/hooks/swap/useSwapInfo.tsx

```diff
@@ -105,6 +106,7 @@ function useComputeSwapInfo(): SwapInfo {
   // Wait until the trade is valid to avoid displaying incorrect intermediate values.
   const slippage = useSlippage(trade)
   const impact = usePriceImpact(trade.trade)
+  const fiatValueChange = useFiatPriceImpact(trade.trade)
```

Comment:
this is a new metric that shouldn't be confused with the 2 above

Reply:
i'd rename this to just `useFiatValueChange`